pydocstringformatter/run.py

```diff
@@ -21,18 +21,37 @@ def __init__(self, argv: Union[List[str], None]) -> None:
         )
         self.config = self._arguments_manager.namespace
 
-        if argv := argv or sys.argv[1:]:
-            self._arguments_manager.parse_options(argv)
-            for formatter in formatting.FORMATTERS:
-                formatter.set_config_namespace(self.config)
-            self._check_files(self.config.files)
-        else:
+        # Display help message if nothing is passed
+        if not (argv := argv or sys.argv[1:]):
             self._arguments_manager.print_help()
+            return
+
+        # Parse options and register on formatters
+        self._arguments_manager.parse_options(argv)
+        for formatter in formatting.FORMATTERS:
+            formatter.set_config_namespace(self.config)
 
-    def _check_files(self, arguments: List[str]) -> None:
+        self._check_files(self.config.files)
+
+    def _check_files(self, files: List[str]) -> None:
         """Find all files and perform the formatting."""
-        filepaths = utils._find_python_files(arguments, self.config.exclude)
-        self._format_files(filepaths)
+        filepaths = utils._find_python_files(files, self.config.exclude)
+
+        is_changed = self._format_files(filepaths)
+
+        if not is_changed:  # pylint: disable=consider-using-assignment-expr
+            if len(filepaths) > 1:
+                files_string = f"{len(filepaths)} files"
+            else:
+                files_string = "1 file"
```

Comment:
```suggestion
                files_string = f"'{filepaths[0].name}'"
```
Maybe a nice little bonus ?

Reply:
Fails with `IndexError` in some test cases. I'll keep it in mind for a future enhancement.